contracts/0.8.9/LidoExecutionLayerRewardsVault.sol

```diff
@@ -73,7 +73,9 @@ contract LidoExecutionLayerRewardsVault {
       * @dev execution layer rewards may be sent as plain ETH transfers
       */
     receive() external payable {
-        emit ETHReceived(msg.value);
+        if (msg.value > 0) {
```

Comment:
Let's don't do that
We aren't going to redeploy the EL Vault unless a critical issue discovered

Reply:
yep, it's low, remove last commit